### PR TITLE
#948 Add job type to elasticsearch logs

### LINKED ISSUE #948: Add job type to elasticsearch logs
When debugging it would be ideal to have the job_type id associated with logs in elasticsearch to help pinpoint potential errors.

### PR DESCRIPTION
**Pull request by Fizz11**
added job_type to logstash output. 
added job_type to main_task_tag, separated by '++' and split out within logstash.conf-template
fixes #948

### CODE REVIEW

File: dockerfiles/logstash/logstash.conf-template [hunk 1]
```diff
@@ -17,14 +17,14 @@ filter {
   if [@metadata][DEBUG] != 'true' {
     ruby {
       init => "@ordernum = 0"
-      code => "@ordernum += 1; event['scale_order_num'] = @ordernum; event['scale_task'] = event['program'].sub(%r{^docker/}, ''); event['scale_job_exe'] = event['scale_task'].sub(%r{_[^_]*$}, ''); event['scale_node'] = event['logsource']; event['stream'] = event['severity'] == 3 ? 'stderr' : 'stdout'"
-      remove_field => ["host", "priority", "timestamp8601", "logsource", "program", "pid", "severity", "facility", "timestamp", "facility_label", "severity_label"]
+      code => "@ordernum += 1; tag_items = event['program'].split('++'); event['scale_order_num'] = @ordernum; event['scale_task'] = tag_items[0].sub(%r{^docker/}, ''); event['scale_job_exe'] = event['scale_task'].sub(%r{_[^_]*$}, ''); event['scale_node'] = event['logsource']; event['stream'] = event['severity'] == 3 ? 'stderr' : 'stdout'; event['job_type'] = tag_items[1]"
+     remove_field => ["host", "priority", "timestamp8601", "logsource", "program", "pid", "severity", "facility", "timestamp", "facility_label", "severity_label", "job_type"]
```
Comment (gisjedi) [inline dockerfiles/logstash/logstash.conf-template]: Looks like an indentation inconsistency here.

Comment (gisjedi) [reply dockerfiles/logstash/logstash.conf-template]: With this change we are going to need to be a bit more careful about maintaining parity between the build of `scale` and `scale-logstash` images. I'd recommend we accomplish this by implementing the following behavior:

* Update `bootstrap.py` to assume a default `LOGSTASH_DOCKER_IMAGE` unless set to a non-empty string. The default should be based off of `MARATHON_APP_DOCKER_IMAGE` with `repo/scale:tag` updated to be `repo/scale-logstash:tag`

**Comment by gisjedi** [comment]
@dfaller This is ready to go, but I wanted your take on it. @Fizz11 has added logic in the bootstrap to ensure there is consistency in the `scale` and `scale-logstash` image tags, which I believe should eliminate drift between what the logstash is expecting and Scale is providing via the Docker tag. I'm aware this is short term, and we need to look for long-term solutions utilizing the DCOS logging subsystem.

**Comment by dfaller** [comment]
Yeah, it's ugly but it will work short term.

**Pull request merged**